pyproject.toml

```diff
@@ -47,17 +45,14 @@ classifiers = [
 
 dependencies = [
     "xlib==0.21",
-    "setuptools<=77.0.3",
     "bscearth.utils<=0.5.2",
     "requests<=2.32.3",
     "networkx<=2.6.3",
     "portalocker<=2.7.0",
     "paramiko>=3.5.0",
     "pyparsing==3.2.1",
     "matplotlib<=3.8.3",
-    "packaging<=24.2",
     "typing_extensions==4.*",
-    "typing<=3.7.4.3",
```

Comment:
Typing is for compatibility with older versions of Python, safe to remove as we are using Python 3.9 now, and typing has been part of stdlib for a while.